src/CommandLineParser.ts

```diff
@@ -32,6 +32,13 @@ export const optionDeclarations: YargsOptions = {
         describe: "Specify if a header will be added to compiled files.",
         type: "boolean",
     },
+    hoisting: {
+        alias: "h",
```

Comment:
h is usually help, maybe choice something different or don't set an alias at all.

Reply:
good point